packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -313,11 +313,11 @@ abstract class FlutterDriver {
   /// See also:
   ///
   ///  * [getWidgetDiagnostics], which gets the [DiagnosticsNode] of a [Widget].
-  Future<Map<String, Object>> getRenderObjectDiagnostics(
+  Future<Map<String, dynamic>> getRenderObjectDiagnostics(
```

Comment:
Why not Object anymore?

Reply:
Here and below.

Reply:
Over aggressive find and replace. I tried making more things not dynamic and they weren't ready to change. Changing back.